__tests__/tests/components/CommentInput.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {Platform} from 'react-native';
+
+import {CommentInput} from 'components';
+
+describe('<CommentInput />', () => {
+
+  const defaultProps = {
+    users: [],
+    userHasPushPermission: true,
+    issueLocked: false,
+    locale: '',
+    onSubmit: () => {},
+  };
+
+  it('should render TextInput and TouchableOpacity if I can post', () => {
+
+    const wrapper = shallow(
+      <CommentInput {...defaultProps}/>
+    );
+
+    expect(wrapper.find('TextInput').length).toEqual(1);
+    expect(wrapper.find('TouchableOpacity').length).toEqual(1);
+    expect(wrapper.find('Text').length).toEqual(0);
+  });
+
+  it("should not render TextInput and TouchableOpacity if I can't post", () => {
```

Comment:
Use 3rd person. `'should not render TextInput and TouchableOpacity if user does not have post permissions'`